machAeroTutorials/overset_theory.rst

```diff
@@ -0,0 +1,158 @@
+
+.. note:: As the solver has to interpolate in the overlapping region, the calculated solution will locally not be
+          as accurate. This means it should not happen in critical regions, like the wing tip.
+
+Implicit Hole Cutting (IHC)
+===========================
+When there are overlapping meshes, one must decide which cells of one grid should exchange information 
+with cells from an other grid. Additionally, there will be some cells that should be ignored at all. 
+This process is called hole cutting. For some solvers, the user must set it up in advance. ADFlow 
+does this implicitly without any additional input. It works on the assumption, that the cells closer 
+to a wall, have a smaller volume. If there are overlapping meshes, it basically uses the smaller cells and 
+blanks/interpolates the bigger ones.
+
+.. figure:: images/overset_IHC.png
+    :align: center 
+
+    Before (left) and after IHC (right).
+..
+    src: overset_guide.pdf page 7
+
+More about IHC can be found here: `Implicit Hole Cutting - A New Approach to Overset Grid Connectivity
+<https://arc.aiaa.org/doi/10.2514/6.2003-4128>`_\.
+
+
+Zipper Mesh
+===========
+As seen in the first figure on this page, there can be multiple nearfield meshes that overlap on a surface.
+This makes it hard to correctly integrate the forces and moments acting there. For that reason, ADFlow uses
+zipper meshes to provide a watertight surface. 
+
+.. figure:: images/overset_zipper.png
+    :align: center 
+
+    Overlapped meshes (left), Removed overlaps (mid), Triangulated gaps (right)
+
+..
+    src: overset_guide.pdf page 20
+
+More about zipper meshes can be found here: `Enhancements to the Hybrid Mesh Approach to
+Surface Loads Integration on Overset Structured Grids 
+<https://www.nas.nasa.gov/assets/pdf/staff/Chan_W_Enhancements_to_the_Hybrid_Mesh_Approach_to_Surface_Loads_Integration_on_Overset_Structured_Grids.pdf>`_\.
+
+
+Things to note
+==============
+Overset meshes can be tricky. Here are a few tips to help you.
+
+Tip #1
+------
+Make sure there is sufficient overlap between meshes.
+
+.. figure:: images/overset_tip1.png
+    :align: center 
+
+    Overlapping is needed between meshes.
+
+Tip #2
+------
+Match cell sizes of the overlapped meshes, especially near boundaries
+
+.. figure:: images/overset_tip2.png
+    :align: center 
+
+    Left: Not recommended. May give a valid hole cutting with additional effort. Right: Better transition. Easier to find interpolation stencils.
+
+Tip #3
+------
+Match the growth ratios of the mesh extrusion. 
+
+* Use similar values of initial cell height for all meshes (``s0`` option in pyHyp)
+* Make sure that all meshes have similar growth ratios during the pyHyp extrusion. A variation of +- 0.05 is okay
+* If you want to priorize one mesh, use slightly smaler values for ``s0`` and growth ratio.
+
+Debugging an Overset Mesh
+=========================
```

Comment:
This section looks super helpful! Could you just briefly mention /  add links to better understand the flooding and orphan cells concepts? Just a couple of sentences should do the job.

Reply:
I will try. I just copied a presentation about this, that Anil gave me. I never really bothered to understand it.

Reply:
https://mdolab.engin.umich.edu/bibliography/Kenway2017a.html

Here is the paper that talks about the details

Reply:
Never mind, I see that you have already done so.